test/cppsuite/test_harness/runtime_monitor.h

```diff
@@ -182,14 +193,78 @@ class db_size_statistic : public statistic {
     int64_t _limit;
 };
 
+class postrun_statistic_check {
+    public:
+    explicit postrun_statistic_check(configuration *config)
+    {
+        const auto config_stats = config->get_list(POSTRUN_STATISTICS);
+        /*
+         * Each stat in the configuration is a colon separated list in the following format:
+         * - Stat name
+         * - Min limit
+         * - Max limit
+         */
+        for (const auto &c : config_stats) {
+            auto stat = split_string(c, ':');
+            if (stat.size() != 3)
+                testutil_die(-1,
+                  "runtime_monitor: Each postrun statistic must follow the format of "
+                  "\"stat_name:min_limit:max_limit\". Invalid format \"%s\" provided.",
+                  c.c_str());
+            const int min_limit = std::stoi(stat.at(1)), max_limit = std::stoi(stat.at(2));
+            if (min_limit > max_limit)
+                testutil_die(-1,
```

Comment:
EINVAL here as well if we decided to use this instead of -1

Reply:
fe68600